fastchat/model/model_adapter.py

```diff
@@ -189,11 +189,15 @@ def load_model(
                 revision=revision,
             )
     elif gptq_config and gptq_config.wbits < 16:
-        return load_gptq_quantized(
+        model, tokenizer = load_gptq_quantized( 
             model_path,
             gptq_config,
             device=device,
         )
+        import accelerate 
+        device_map = accelerate.infer_auto_device_map(model, max_memory=kwargs["max_memory"], no_split_module_classes=["LlamaDecoderLayer"]) 
```

Comment:
can we only enable this when `num_gpus` > 1?

Reply:
Moved logic for device dispatch from modules/gptq.py for single device dispatch to model_adapter.py, and added selection logic for single or multi gpu dispatching.